### ISSUE TITLE
PMD part disabled for JDK 11

### ISSUE THREAD

**Issue by AdamCarroll** [Santulator/Santulator#2]
The PMD build has been partly disabled for JDK 11 because when enabled it fails with false positives for unused imports.  The build also produces errors of the following form:
```
Error during type resolution due to: java.lang.NoClassDefFoundError: javafx/beans/value/ObservableBooleanValue
```

When these problems have been resolved, PMD should be re-enabled for all parts of the JDK 11 build.

**Comment by ColColonCleaner** [comment]
@AdamCarroll Has this issue been logged with PMD? I'm running into a very similar issue.

**Comment by ColColonCleaner** [comment]
@AdamCarroll I located the issue on my end for the NoClassDefFoundError, so the solution might help you too. Both the taskdef's classpathref and the auxClasspath need to contain all of the jars you want to reference. In older versions it appears that PMD used some combination of the two but now those requirements have changed.

**Comment by AdamCarroll** [comment]
The PMD is temporarily disabled in the GUI sub-project using the following:
https://github.com/Santulator/Santulator/blob/017def11aef0527dfc0ba74e57a435b8e1ccdc22/gui/build.gradle#L109-L110

**Comment by AdamCarroll** [comment]
This is fixed in ea1a46ae4b9ef166ae042acb5794d3ca8be107e4.  Thanks @ColColonCleaner for the top about the classpath.  You're a star! :star:
